op_crates/fetch/lib.rs

```diff
@@ -46,7 +51,10 @@ pub fn init(isolate: &mut JsRuntime) {
   // workspace root.
   let display_root = manifest_dir.parent().unwrap().parent().unwrap();
   for file in files {
-    println!("cargo:rerun-if-changed={}", file.display());
+    if build_time {
+      println!("cargo:rerun-if-changed={}", file.display());
+    }
```

Comment:
If we're initializing at runtime, ideally we'd use include_str for the source code...